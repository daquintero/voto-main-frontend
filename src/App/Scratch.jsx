// Main Scratch Indexing Container for all of the Routes
import React from 'react';
import Page404 from '../shared/components/404/Page';

const Scratch = () => (
  <div className="layout">
    <Page404 />
  </div>
);

export default Scratch;
