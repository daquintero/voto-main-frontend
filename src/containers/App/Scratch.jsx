// Main Scratch Indexing Container for all of the Routes
import React from 'react';

// Each Compendium

const Scratch = () => (
  <div className="layout">
      Hello adsfadsf a
  </div>
);

export default Scratch;
