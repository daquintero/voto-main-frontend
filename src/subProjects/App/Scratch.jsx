// Main Scratch Indexing Container for all of the Routes
import React from 'react';
import { Container } from 'reactstrap';

// TODO FINISH PAGE TEST
import VariableGrid from '../Reusable/Grid/components/VariableGrid';

// Test Sample Data
const data = [{
  id: 2,
  type: 'corruption.CorruptionCase',
  size: 'full',
  title: 'Whatsup asdn amsda jasd nas dn asnd ',
  date: new Date(),
  statistics: [{
    id: 3,
    link: 'jjj',
    title: 'informacion',
    data: '10',
    icon: 'user',
  }],
}, {
  id: 6,
  type: 'corruption.InformativeSnippet',
  size: 'full',
  title: 'Whatsup asdn amsda jasd nas dn asnd ',
  date: new Date(),
  statistics: [{
    id: 3,
    link: 'jjj',
    title: 'informacion',
    data: '10',
    icon: 'user',
  }],
}, {
  size: 'full',
  title: 'Alguna Information asdfasd ',
  type: 'Employment',
  description: 'Descripcion',
  organization: 'Some organization',
}, {
  size: 'full',
  title: 'Alguna Information asdfasd ',
  type: 'Employment',
  description: 'Descripcion',
  organization: 'Some organization',
},
];

const Scratch = () => (
  <div className="layout">
    <Container>
      <VariableGrid data={data} />
    </Container>
  </div>
);

export default Scratch;
