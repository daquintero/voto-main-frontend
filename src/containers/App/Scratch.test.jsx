import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Scratch from './Scratch';

it('Scratch renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<Scratch />, div);
  ReactDOM.unmountComponentAtNode(div);
});

it('Scratch renders correctly without crashing', () => {
  const tree = renderer
    .create(<Scratch />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
