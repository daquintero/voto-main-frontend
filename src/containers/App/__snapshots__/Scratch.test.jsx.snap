// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Scratch renders correctly without crashing 1`] = `
<div
  className="layout"
>
  Hello adsfadsf a
</div>
`;
